src/components/WebexMeeting/WebexMeeting.js

```diff
@@ -0,0 +1,46 @@
+import React, {Fragment} from 'react';
+import PropTypes from 'prop-types';
+import {Spinner} from '@momentum-ui/react';
+
+import {useMeeting} from '../hooks';
+
+import {WebexInMeeting, WebexInterstitialMeeting, WebexMeetingControl, WebexMeetingControls} from '..';
+
+import './WebexMeeting.css';
+
+/**
+ * Webex Meeting component displays all meeting contents.
+ *
+ * @param {object} props
+ * @returns {object} JSX of the component
+ */
+export default function WebexMeeting({meetingDestination, setControls}) {
+  const {ID, remoteVideo} = useMeeting(null, meetingDestination);
+
+  const isActive = remoteVideo !== null;
+
+  const controls = setControls(isActive).map((key) => <WebexMeetingControl type={key} key={key} />);
+  const meeting = ID ? (
+    <Fragment>
+      {isActive ? <WebexInMeeting meetingID={ID} /> : <WebexInterstitialMeeting meetingID={ID} />}
+      <div className="controls">
+        <WebexMeetingControls meetingID={ID}>{controls}</WebexMeetingControls>
+      </div>
+    </Fragment>
+  ) : (
+    <Spinner />
+  );
+
+  return <div className="meeting">{meeting}</div>;
+}
+
+WebexMeeting.propTypes = {
+  meetingDestination: PropTypes.string.isRequired,
+  setControls: PropTypes.func,
+};
+
+WebexMeeting.defaultProps = {
+  // eslint-disable-next-line no-confusing-arrow
+  setControls: (isActive) =>
```

Comment:
Please add a `func` typedef explaining this function prop

Reply:
not sure how you want it.